### PR TITLE
Dark and light mode

### PR DESCRIPTION
**Pull request by YanxunLi**
Note:
1. logout will switch back to default mark mode.
2. The color of button images can’t be switched.

### CODE REVIEW

File: extension/JavaScript/leaderboard.js [hunk 1]
```diff
@@ -74,18 +74,48 @@ function sortByPoints(){
...
    reverse = !(rankByPoints || reverse);
    rankByPoints = false;
    useJSON(response);
 }
 
+/*========================================================================
+Functionality of 'dark' button --> Interchange between dark and light mode
+========================================================================*/
+function switchMode() {
+   console.log("Button Clicked");
```
Comment (winsonluk) [inline extension/JavaScript/leaderboard.js:87]: nit: remove console.log

File: extension/JavaScript/task.js [hunk 2]
```diff
@@ -115,3 +115,16 @@ for(let i = 0; i < myValues.length; i++) {
    newNode.innerHTML = myValues[i];
    output.appendChild(newNode);
 }
+
+/*==============================================================
```
Comment (winsonluk) [inline extension/JavaScript/task.js:119]: ought: repeated code...could you make a separate JS file and import it here instead? same for tasks.js, team.js, userProfile.js, users.js, etc.

File: extension/JavaScript/leaderboard.js (hunk 1 above)
Comment (YanxunLi) [reply extension/JavaScript/leaderboard.js]: removed.

File: extension/JavaScript/task.js (hunk 2 above)
Comment (YanxunLi) [reply extension/JavaScript/task.js:121]: refactored

**Pull request merged**